content/community/tools-jsx.md

```diff
@@ -1,53 +1,54 @@
-* **[gradle-react-plugin](https://github.com/ehirsch/gradle-react-plugin):** Transform jsx sources during a gradle build.
-* **[grunt-react](https://npmjs.org/package/grunt-react):** [GruntJS](https://gruntjs.com/) task.
-* **[gulp-react](https://npmjs.org/package/gulp-react):** [GulpJS](https://gulpjs.com/) plugin.
-* **[brunch-react](https://www.npmjs.org/package/react-brunch):** [Brunch](https://brunch.io/) plugin.
-* **[jsx-requirejs-plugin](https://github.com/philix/jsx-requirejs-plugin):** [RequireJS](https://requirejs.org/) plugin.
-* **[react-meteor](https://github.com/benjamn/react-meteor):** [Meteor](https://www.meteor.com/) plugin.
-* **[pyReact](https://github.com/facebook/react-python):** [Python](https://www.python.org/) bridge to JSX.
-* **[react-rails](https://github.com/facebook/react-rails):** Ruby gem for using JSX with [Ruby on Rails](https://rubyonrails.org/).
-* **[react-laravel](https://github.com/talyssonoc/react-laravel):** PHP package for using ReactJS with [Laravel](https://laravel.com/).
-* **[ReactJS.NET](https://reactjs.net/):** .NET library for React and JSX.
-* **[sbt-reactjs](https://github.com/ddispaltro/sbt-reactjs)** SBT/Play/Scala JSX compiler plugin
-* **[mimosa-react](https://github.com/dbashford/mimosa-react):** [Mimosa](http://mimosa.io) plugin.
-* **[react-grails-asset-pipeline](https://github.com/peh/react-grails-asset-pipeline):** Assets for react and precompilation of jsx files in [Grails](https://grails.org/).
-* **[gore-gulp](https://github.com/goreutils/gore-gulp):** Wrapper around [webpack](https://webpack.github.io/), [eslint](https://eslint.org/), [mocha](https://mochajs.org/) for ease of use and zero configuration.
-* **[webpack](https://github.com/webpack/webpack):** Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jade, coffee, css, less, ... and your custom stuff.
-* **[webpack-bbq](https://github.com/wenbing/webpack-bbq):** transform your src to lib, supports server rendering and static rendering.
-* **[jsxtransformer](https://github.com/cronn-de/jsxtransformer):** Compile pipeline for jsx files in Java
-* **[babylon-jsx](https://github.com/marionebl/babylon-jsx)**: Transform JSX to ES2015 with babylon sans babel
-* **[CRA Universal CLI](https://github.com/antonybudianto/cra-universal)** - A simple CLI to create and build Express server for your create-react-app projects, featuring Server-side rendering and Code-splitting.
+## Outils de construction {#build-tools}
+
+* **[Create React App](https://github.com/facebookincubator/create-react-app)** : une façon **officielle** de créer des applications React sans configuration.
+* **[nwb](https://github.com/insin/nwb)** : boîtes à outils pour les applications React, PReact & Inferno et d'autres modules npm pour le web, sans configuration (tant que vous n'en avez pas besoin).
```

Comment:
```suggestion
* **[nwb](https://github.com/insin/nwb)** : une boîte à outils pour les applications React, Preact & Inferno et d'autres modules npm pour le web, sans configuration (jusqu’à e que vous en ayez besoin).
```

Reply:
Petit oubli.
`jusqu’à e que` ➡️ `jusqu’à ce que`

Reply:
C'est ajusté mais j'ai pas les droits pour le committer moi-même sur ta PR (ça va gêner pour le *final tweaks* éventuel, d'ailleurs ; tu as mis "allow push access to maintainers" dessus ?)

Reply:
Oui, comme pour les autres PR :
![image](https://user-images.githubusercontent.com/623217/55890823-b0809080-5bb3-11e9-9ac3-9402a2651213.png)